interactive-app/simple-viewer-app/test/end-to-end/setupTests.ts

```diff
@@ -10,14 +10,16 @@ export let page: Puppeteer.Page;
 export let browser: Puppeteer.Browser;
 
 beforeEach(async () => {
-  let launchOptions: Puppeteer.LaunchOptions = { dumpio: true }; // , headless: false };
+  let launchOptions: Puppeteer.LaunchOptions = { dumpio: true, headless: false };
```

Comment:
Did you mean to leave this set to `false`?

Reply:
For some reason, the tests load very very slowly in headless. I believe it has to do with headless not having GPU access, so the iModel loads slowly. The tests worked headless after 20+ minutes, but that is ridiculous. A few seconds with a head.

Reply:
Ah okay that makes sense.  It probably goes into software rendering mode which would definitely take a while.